tool/expose_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Can we make this script more readable by using dart / python ? Would make it easier for maintainability :-)

Reply:
Great direction and thought process!

Reply:
Thanks @pavanpodila !

I didn't know the way you would have liked to go, that's why I created it in "simple" bash.

I will attempt a dart port if you are ok with it.

Reply:
Your bash is way better than mine :-). Didn't look simple at all.

Ya dart would be good!